telluric/georaster.py

```diff
@@ -1040,31 +1036,34 @@ def reproject(self, new_width, new_height, dest_affine, dtype=None, dst_crs=None
 
         return new_raster
 
-    def to_png(self, transparent=True, thumbnail_size=None, resampling=None, stretch=False):
+    def to_png(self, transparent=True, thumbnail_size=None, resampling=None, in_range='dtype', out_range='dtype'):
```

Comment:
I think the in_range should be `image` and not `dtype` so the resulting `png` file will look better.
did you check that the resulting `png` file is the same as before?

Reply:
Sorry I didn't notice `stretch` is false by default, so leave it `dtype`

Reply:
1. Yes I did. In the current code `False` is the default value of `stretch` in `to_png` function: https://github.com/satellogic/telluric/blob/186365ecf4f2acb8cd08be7a3c3ca5991bc97e89/telluric/georaster.py#L997 And so for preserving such behavior I'm using 'dtype' as default value for `in_range` and `out_range`.

2. But for showing raster with jupyter notebook `stretch=True`:
https://github.com/satellogic/telluric/blob/186365ecf4f2acb8cd08be7a3c3ca5991bc97e89/telluric/georaster.py#L1091-L1093 In this case I'm using `in_range='image'` which means the same behavior.